docker/transport/basehttpadapter.py

```diff
@@ -6,3 +6,15 @@ def close(self):
         super().close()
         if hasattr(self, 'pools'):
             self.pools.clear()
+
+    # Hotfix for requests 2.32.0 and 2.32.1: its commit
+    # https://github.com/psf/requests/commit/c0813a2d910ea6b4f8438b91d315b8d181302356
+    # changes requests.adapters.HTTPAdapter to no longer call get_connection() from
+    # send(), but instead call _get_connection().
+    def _get_connection(self, request, *args, proxies=None, **kwargs):
```

Comment:
I'll be yanking 2.32.0 and 2.32.1 once we get 2.32.2 out so this can probably be omitted from the patch.

Reply:
I've removed these lines. (For everyone else: now 2.32.2 is out, and 2.23.0 and 23.2.1 have been yanked.)

Reply:
you mean 2.32.0 and 2.32.1 yanked? I still got it installed as an indirect pip dependency just an hour ago.

Reply:
Caching perhaps. You can see here they are yanked: https://pypi.org/project/requests/#history.